pa5/views.sql

```diff
@@ -0,0 +1,29 @@
+﻿CREATE VIEW info_about_books AS
+SELECT b.title AS book_title,
+       GROUP_CONCAT(a.name ORDER BY a.name SEPARATOR ', ') AS author_name,
+       CONCAT(p.name, ', ', b.release_year) AS publisher_release_year,
+       l.loan_date,
+       l.left_days
+FROM book b
+
+         INNER JOIN author2book ab ON b.id = ab.book_id
+         INNER JOIN author a ON a.id = ab.author_id
+         INNER JOIN publisher p ON p.id = b.publisher_id
+
+         INNER JOIN (SELECT l.book_id,
+                            DATEDIFF(l.due_date, CURDATE()) AS left_days,
+                            DATE_FORMAT(loan_date, '%b, %d, %Y') AS loan_date
+                     FROM loan l) l ON b.id = l.book_id
```

Comment:
Not a bug but it is not recommended to use the same alias for the different tables or subqueries.